### PR TITLE
Prevent PanelBody title from being overlapped by arrow

### LINKED ISSUE #29913: PanelBody arrow may overlap its title
## Description
The German translation for “[Ordered list settings](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/list/ordered-list-settings.js#L10)” is longer than the available space in the PanelBody title area but not long enough to wrap into a new line.

## Step-by-step reproduction instructions
1. Set your WordPress to German.
2. Create a list block.
3. Switch list to ordered list.
4. Look into the block sidebar.

## Expected behaviour
Arrow does not overlap its title.

## Actual behaviour
Title may get overlapped by the PanelBody arrow.

## Screenshots or screen recording (optional)
Bildschirmfoto 2021-03-16 um 20 52 29

## Proposed solution

Add a `padding-right: 48px;` to the `.components-panel__body-toggle.components-button` [here](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/panel/style.scss#L79) to add a restricted zone for the title.

Bildschirmfoto 2021-03-16 um 21 18 16

## WordPress information
- WordPress version: 5.7
- Gutenberg version: Not installed
- Are all plugins except Gutenberg deactivated? No
- Are you using a default theme (e.g. Twenty Twenty-One)? No

## Device information
- Device: Desktop
- Operating system: macOS 11.2.3
- Browser: Safari 14.0.3 (16610.4.3.1.7)

### PR DESCRIPTION
**Pull request by MatzeKitt**
## Description
This will fix #29913.

## How has this been tested?
* Checked the updated style in Safari, Chrome and Firefox.

## Types of changes
Non-breaking styling fix for PanelBody title area.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by talldan** [comment]
Thanks for working on this @MatzeKitt. 

Gutenberg now uses `trunk` as its main branch, so I've switched the base branch in the PR. There are some failing tests because of an issue with the docker environment, bit this has been fixed in a commit to `trunk`. Would you be able to rebase your branch on top of trunk to get the tests passing? (`git pull --rebase origin trunk` should do the trick).

**Comment by MatzeKitt** [comment]
@talldan Yes, I will try! Thank you for the information.

File: packages/components/src/panel/style.scss [hunk 1]
```diff
@@ -77,6 +77,7 @@
 .components-panel__body-toggle.components-button {
 	position: relative;
 	padding: $grid-unit-20;
+	padding-right: 48px;
```
Comment (ntsekouras) [inline packages/components/src/panel/style.scss]: Thank you for your contribution @MatzeKitt !

I think you should make your change in the above `padding` rule. Also it's better to use [GB declared variables](https://github.com/WordPress/gutenberg/blob/4e9aa5dc71d8bdef4ee7b7165d8a8704e28ef657/packages/base-styles/_variables.scss#L39).
`$grid-unit-60` equals `48px`.

Comment (MatzeKitt) [reply packages/components/src/panel/style.scss]: I changed my pull request accordingly. 🙂

**Review by talldan — APPROVED** [review]
Nice improvement, thanks @MatzeKitt.

**Comment by ntsekouras** [comment]
Congrats on your first contribution in GB @MatzeKitt ! 🎉  Thank you!

**Comment by MatzeKitt** [comment]
Thank you! 🙂

**Pull request merged**